test/remote/gateways/remote_priority_test.rb

```diff
@@ -5,6 +5,7 @@ def setup
     @gateway = PriorityGateway.new(fixtures(:priority))
 
     @amount = 2
+    @credit_amount = -2000
```

Comment:
Should the @credit_amount be negative? the refund tests all have positive integers as the @amount and are passing. If you are crediting a negative number that may have an unintended consequence.

Reply:
Yes it should be negative that is how our system works.